src/config.ts

```diff
@@ -54,6 +54,10 @@ export default {
       description:
         "You can assign this role to yourself to be pinged when there are updates regarding community battlesnake games or Eddie' video for battlesnake",
     },
+    MLH_GUILD: {
+      name: 'mlh_guild',
+      description:
+        'You can assign this role to yourself if you are participating with our guild in an MLH LHD Share hackathon.',
```

Comment:
Are these the only kind of hackathons we do?

Reply:
if participating as a guild , yes for now. other hackathons are not as per guild. This happens in LHD only

Reply:
Perfect - thanks for clarifying!